### DISCUSSION TITLE
Reactions do not trigger any webhook

### DISCUSSION THREAD

**Discussion by jake-nz** [community/community#7168] [Projects and Issues]
We would like our app to take immediate action when a user adds a reaction to a comment. It seems that we have no way to know that this has happened, other than polling the API periodically to check for new reactions.

**Comment by elviskahoro** [comment]
Cross posting from https://github.com/github-community/community/discussions/20824#discussioncomment-3208044:
Huge +1. Articulation for why this is important: People add reactions to issues as a way of expressing interest. To my knowledge there isn't a way to query the list of people who've subscribed to an issue and so instead we ask folks to add an emoji. This is better than asking them to comment because for issues with a large amount of interest, when people comment: "+1" it spams the inboxes of those currently subscribed.

**Reply by elviskahoro** [reply]
@queenofcorgis

**Comment by tim-finnigan** [comment]
Also +1 but specifically for adding a webhook for reactions on issues (in addition to comments).

**Comment by LucBerge** [comment]
+1 I also need this webhook for comment on issues

**Comment by enoreyes** [comment]
+1 also interested in this

**Reply by enoreyes** [reply]
For more context, when building automated systems that interact with github, you might want users to be able to make a decision or trigger an action in github on the issue. Right now, you have to continuously poll things you are listening to.

**Comment by whoamitty** [comment]
+1 Me too I need it badly
for manage project
and automate move issue on Done
according to reactions

**Comment by ghostinhershell** [comment]
Hey there @jake-nz and all 👋🏾 

Thanks so much for reaching out in our community, and for supporting what would be a very cool and useful addition to our webhooks. I took a look and do see that a feature request already exists for this, so I've gone ahead and added you guys' vote to that feature request. I'll look for other ways to signal boost this internally as well.

**Reply by elviskahoro** [reply]
@ghostinhershell thanks in advance! This has been a request for awhile! any progress is appreciated

**Reply by doallthethings** [reply]
Hi @ghostinhershell I had someone bring up this issue at Developer Week. I was wondering if we could provide a status with respect to planned or not. I imagine this is something we will eventually get to but likely not planned on the current roadmap. Is my assumption correct?

**Reply by elviskahoro** [reply]
Thanks @doallthethings !

**Comment by elviskahoro** [comment]
For context @ghostinhershell @doallthethings 

We have nearly 5000 GitHub issues and we can get 400+ upvotes on a particular issue in the span of only a couple of weeks:
https://github.com/warpdotdev/Warp/issues/4240

If we had a webhook event, it would dramatically improve our ability to measure and graph what our community wants prioritized. I could poll each issue one at a time periodically but this doesn't scale well given that I'd have to do this for thousands of issues EVERY week--issues which are also only evergrowing!

![CleanShot 2024-03-05 at 11 03 33@2x](https://github.com/community/community/assets/29553206/118b812a-d2a4-470c-adaa-54d15b1f1861)

**Reply by LucBerge** [reply]
I personnaly still don't understand why github don't have a native way to prioritize issues based on user votes... Just a button at the top of each issue :"Upvote". Then you can sort issues by "Upvote"

**Comment by enoreyes** [comment]
Seconded to @LucBerge and @elviskahoro. This issue is really important for prioritization as well as receiving feedback. Polling every instance is very expensive / unreasonable. Our github application posts thousands of comments a week which each would need to be polled at least daily to capture events.

Any updates / timeline for this feature would be appreciated!

**Comment by dimmyjr-nu** [comment]
+1

**Comment by jr-k** [comment]
+1

**Comment by arvi18** [comment]
+1

**Comment by zbrown001** [comment]
+1

**Comment by mrtrvl** [comment]
+1

**Comment by BaherW** [comment]
+1

**Comment by aryairani** [comment]
+1

**Comment by ristiisa** [comment]
+1

**Comment by JoshuaKGoldberg** [comment]
This is a blocker for https://github.com/octoguide/bot/issues/32: having standard actions runs take action based on emoji reactions. For example, a _"all past violations now pass"_ comment from a checker delete itself when 👍 reacted to.
